client/src/panels/panel_declarations/finances/welcome_mat/welcome_mat.js

```diff
@@ -477,12 +477,6 @@ const WelcomeMat = (props) => {
       />
     );
   } else if (type === "hist_estimates") {
-    //active, non-DP org, CR or program
-    //has no FTEs
-
-    //full-width,
-    //5 yrs ago, last year, this year, graph
-    //text about hist-diff
```

Comment:
This wasn't a comment about the `full-width` class name, this is just a generic description of the welcome mat's `type === "hist_estimates"` case. Not the kind of documentation I'd normally want in comments (should be clearer in the code/naming of things, would have to double check that these comments are still up to date with reality). The welcome mat is a huge mess though, so any additional help from comments (until it's cleaned up) is good to have.